### ISSUE TITLE
a question about 302_classfication.py

### ISSUE THREAD

**Issue by RichardcLee** [MorvanZhou/PyTorch-Tutorial#70]
why do not you use F.softmax in follow code?

code: _prediction = torch.max(out, 1)[1]_

![canvas](https://user-images.githubusercontent.com/28611665/51439932-db1d2480-1cfb-11e9-8466-1d1262cf771e.png)

**Comment by RichardcLee** [comment]
> 
> 
> You could use softmax instead max and sample from the softmax. However, if you only need the maximum value in the `out`, calculating softmax results in an additional computation out of max.

thanks
